### DISCUSSION TITLE
Any plans for music library support?

### DISCUSSION THREAD

**Discussion by zpz5HAU-tgc3fgw2xwr** [meisnate12/Plex-Meta-Manager#188] [Ideas]
Been using this project a lot the past few days, really liking it so far. It seems to cover just about everything I need it for. As the title says, are there any plans for music library support?

**Comment by meisnate12** [comment]
im sure i could but the biggest problem is I don't use plex for music so I don't really have much testing capabilities there but make an issue requesting it and ill try to get to it

**Reply by Critical-Impact** [reply]
I have quite a large library so if you need any testers I'm down to give it a go.

**Reply by meisnate12** [reply]
I'll let you know when I get there I haven't really started on trying to add it since I've been working on my smart collections update

**Comment by douginoz** [comment]
Are there music collection lists on websites somewhere?

**Reply by meisnate12** [reply]
thats an excellent question lol but it would definitely still work with plex builders and filters

**Reply by Critical-Impact** [reply]
I'd say the best bet would either be the musicbrainz api or discogs api.

**Reply by DeftNerd** [reply]
I wrote a feature request for TuneFind.com and outlined the API endpoints. It's a website that lists what songs are found in various movies and TV episodes. Seems like a great PMM fit because it could allow people to make collections or playlists of music in their favorite TV shows or movies. 
#938

**Reply by sergiocos** [reply]
About lists and collections: I would do anything to have an auto-collection for these lists: **[The Rolling Stone 500 Greatest Albums of All-Time](https://www.rollingstone.com/music/music-lists/best-albums-of-all-time-1062063/)** or **[1001 Albums You Must Hear Before You Die](https://www.discogs.com/lists/1001-Albums-You-Must-Hear-Before-You-Die-All-editions-combined/991847)** or **[Rolling Stone 500 Best Songs of All-Time](https://www.rollingstone.com/music/music-lists/best-songs-of-all-time-1224767/)**

**Reply by sergiocos** [reply]
I did that manually for Roon, since I can backup and import database, but I'm not gonna do it manually for Plex, too, since I can lose the entire collection on reinstall.
**[foto-1](https://www.dropbox.com/s/oovt9hqafc3a5cy/Screenshot%202022-08-17%2023.57.24.png?dl=0)**, **[foto-2](https://www.dropbox.com/s/w335zl4pxtttv4c/Screenshot%202022-08-17%2023.57.05.png?dl=0)**

**Comment by gitttt54** [comment]
I pers. use Plex mostly for music. Very large library and I have a testing server too. 
If I can help in testing or something else, let me know.
